torch/_export/passes/lift_constants_pass.py

```diff
@@ -15,33 +17,35 @@
 
 
 class ConstantAttrMap(collections.abc.MutableMapping):
-    """A mapping class that understands how to use module constants (tensors and
-    ScriptObjects) as keys. We store tensors normally, but ScriptObjects are
-    stored by hash, because different torch.ScriptObjects can point to the same
-    underlying value (but we guarantee that they will `hash()` to the same value
+    """A mapping class that understands how to use module constants (tensors,
+    ScriptObjects, FakeScriptObjects) as keys. We store tensors and FakeScriptObjects normally,
+    but ScriptObjects are stored by hash, because different torch.ScriptObjects can point to
+    the same underlying value (but we guarantee that they will `hash()` to the same value
```

Comment:
I'm a bit lost, do we still have ScriptObjects here? I thought that only FakeScriptObjects should show up during tracing.